js/symbol/intersection_tests.js

```diff
@@ -0,0 +1,52 @@
+'use strict';
```

Comment:
Can these be consolidated with [`util/intersection_tests.js`](https://github.com/mapbox/mapbox-gl-js/blob/master/js/util/intersection_tests.js)?